pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -213,6 +213,30 @@ self: super: {
     meta.maintainers = with stdenv.lib.maintainers; [ minijackson ];
   };
 
+  LeaderF = super.LeaderF.overrideAttrs(old: {
+    buildInputs = [
+      python3
+      stdenv
```

Comment:
stdenv shouldn't be necessary
```suggestion
      stdenv
```

Reply:
@jonringer, is there a different way to get gcc? The plugin compiles a C extension.

Reply:
oh hmmm. I think for the vimplugins they do a `{ stdenv = stdenvNoCC }`.... I'm not actually sure

Reply:
I mean, it compiled fine for me when I used stdenv 🤷‍♂️

Reply:
Actually, I just tried it without stdenv and it works fine. I'll remove the dependency. Thanks, @jonringer!